### DISCUSSION TITLE
Reverse Engineering - Battery SoC in CAN Sniffer

### DISCUSSION THREAD

**Discussion by mitkodev** [collin80/SavvyCAN#704] [Q&A]
Hey, I've recently started using SavvyCan for reverse engineering and it's definitely great! I need an advice or idea about finding the state of charge of EV, which in the end is just a number 0-100. I tried to use the CAN Sniffer, which works very good, but since there are so many packages and hexIds, it's hard to locate the change of the value for SoC mainly because the changed bytes flashes for less than a second and draining the battery with 1 percent takes 10-15 minutes so it is easy to miss. Am I missing some setting in order to locate it in a easier manner, or if you can help me with better way of finding such changes? Is there a way for the changed bytes to flash for a bit more at least?

**Comment by cdxa1** [comment]
Fast and easy way is to buy a used instruments, record can traces, transmit to instruments „on bench”, isolate ignition frame and play it in the loop, than transmit messages one by one from recorded data. Than catch the moment when soc will be visible. You can also use your instruments :)
